src/sentry/templates/sentry/layout.html

```diff
@@ -53,14 +53,9 @@
   {% endscript %}
 
   {% block scripts %}
-  {% locale_js_include %}
```

Comment:
Removing this from the template and handling this inside of our app code.

Reply:
One step closer to changing locale without a full page refresh 👍